promgen/tasks.py

```diff
@@ -43,6 +43,8 @@ def process_alert(alert_pk):
                 alert.delete()
                 return
 
+    tasks.index_alert.delay(alert.pk)
```

Comment:
```suggestion
    index_alert.delay(alert.pk)
```

`index_alert` already exists in this file, so no need to reference the module